app/domain/authentication/authn_jwt/jwt_configuration_interface.rb

```diff
@@ -0,0 +1,10 @@
+module Authentication
+  module AuthnJWT
+    # Interface containing JWT configuration functions to be implemented in JWTConfiguration class for a vendor
```

Comment:
i can't say that i understand what a `JWTConfiguration class for a vendor` will be but i'm ok with merging this and waiting for the full implementation to understand